src/utils/generated/graphql.tsx

```diff
@@ -61,6 +61,36 @@ export type Query = Node & {
   counters?: Maybe<CountersConnection>;
   /** Reads and enables pagination through a set of `DataTable`. */
   dataTables?: Maybe<DataTablesConnection>;
+  /** Reads and enables pagination through a set of `DataTableActiveIngredient`. */
```

Comment:
Don't want to commit the Angola types to repo.

Reply:
Ok - cool I'll revert this change by using core_templates loaded in